src/eduid_common/config/parsers/etcd.py

```diff
@@ -15,6 +15,8 @@
 
 class EtcdConfigParser(object):
 
+    UPPERCASE = True
```

Comment:
Can't we change this to an argument to read_configuration? That way we only need this class and can remove IdPEtcdConfigParser.

Reply:
Yes done